librocksdb-sys/build.rs

```diff
@@ -226,6 +226,11 @@ fn build_rocksdb() {
         config.define("ROCKSDB_IOURING_PRESENT", Some("1"));
     }
 
+    if std::env::var("CARGO_CFG_TARGET_POINTER_WIDTH").unwrap() != "64" {
```

Comment:
I would propose to use `matches!` macro here rather than `unwrap`:
 ```suggestion
    if matches!(std::env::var("CARGO_CFG_TARGET_POINTER_WIDTH"), Ok(width) if width != "64") {
```

Reply:
The other places also use `unwrap`, so I thunk this would be more consistent

Reply:
I changed it to use `env::var` though, I think this way it looks the cleanest. CARGO_CFG_TARGET_POINTER_WIDTH is [guaranteed to be defined in build scripts](https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-build-scripts), so IMO it's better to have it panic and let us know something went terribly wrong.